recipes/easygdf/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
It looks like the license is included in the PyPI tarball, so you shouldn't add the file manually here.